README.md

```diff
@@ -0,0 +1,66 @@
+# Team Awesome 📅
+
+Team Awesome is a "Software as a Service" (SaaS) solution that can be used by Humanoids employees for planning projects, tracking time and much more. On the timesheet page users can record their worked hours a get a clear overview of their work by filtering by client and sorting by date. On the Team members page users can provide information on the projects and clients they worked on. By clicking the green button on the right top-corner users can fill in the forms to provide all the information.
```

Comment:
There's an incomplete word here: their worked hours a get a clear overview (--> their worked hours and get a clear overview?)